### ISSUE TITLE
--remove-from-playlist Doesn't work

### ISSUE THREAD

**Issue by promasterx** [jrnewell/spotify-ripper#249]
Hello everyone 

As I understood passing argument "--remove-from-playlist" removes track from the playlist that is located in Spotify, but for some reason it doesn't work. 

Upon completion I can see the output `Removing successfully ripped tracks from playlist example`  but when I check the "example" playlist in Spotify it was still there... 

Then I thought, maybe it Spotify Ripper creates local version of playlist, thus passing the argument removes the track from local playlist.... to check this hypothesis I've removed the ripped tracks from folder where it has been downloaded, but then ripper logs-in, tries to load the playlist then logs-out. Very I odd I thought

My second guess was to pass `--playlist-sync` along with `--remove-offline-cache` and `--overwrite` but still didn't work, it kept logging-out. 

On last attempt I deleted content of /.spotify-ripper/Users/ folder, finally ripping started

So could you please help with ability to remove tracks from playlist? It would be great to have the tracks removed from the my "cloud" playlist 

Thanks in advance

Spotify Ripper - v2.9.1
Python v3.5.0

**Comment by filatovvo** [comment]
I have the same problem.

**Comment by jrnewell** [comment]
yeah i see it too.  The songs are removed if you rerun spotify-ripper (so it is changed in the local cache), but it is not persisted to the spotify's servers.  Unfortunately, I think this is a change on Spotify's side - I played with it for a while and couldn't get it sync.   Since Spotify doesn't support libspotify anymore - I don't seeing it being fixed.  I will look into to seeing if it is difficult to use the web API, but I might have disable or put a warning for this option.

**Comment by jrnewell** [comment]
I added a warning to this option for now ``2.10.2``
